docs/reference/cli/filters.md

```diff
@@ -83,6 +90,117 @@ This command will analyze your cluster Vulnerabilities through K8sGPT. Depending
 - Error: critical Vulnerability found ID: CVE-2019-8457 (learn more at: https://avd.aquasec.com/nvd/cve-2019-8457)
 ```
 
+## Prometheus
+
+Activate the [Prometheus](https://prometheus.io) integration:
+```bash
+k8sgpt integration activate prometheus --namespace <namespace>
+```
+
+The Prometheus integration does not deploy resources in your cluster. Instead,
+it detects a running Prometheus stack in the provided namespace.
```

Comment:
Oh in that case, we should mention that people should have a Prometheus installation already running in their cluster

Reply:
Yep - moved this above per the previous comment.